Cargo.toml

```diff
@@ -32,29 +32,30 @@ num-traits = { version = "0.2.14", default-features = false }
 serde = { version = "1.0.119", features = ["derive"] }
 serde_json = "1.0.61"
 url = "2.2.0"
-codec = { package = "parity-scale-codec", version = "1.3.6", default-features = false, features = ["derive", "full"] }
-
-frame-metadata = "12.0.1"
-frame-support = "2.0.1"
-sp-runtime = "2.0.1"
-sp-version = "2.0.1"
-pallet-indices = "2.0.1"
+codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive", "full"] }
+funty = "=1.1.0"
```

Comment:
perhaps add an issue or comment on this because of https://github.com/myrrlyn/funty/issues/3